src/index.html

```diff
@@ -4,9 +4,324 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img
+        src="images/logo.png"
+        alt="MOYO logo"
+        class="logo"
+        >
```

Comment:
```suggestion
        <img
          src="images/logo.png"
          alt="MOYO logo"
          class="logo"
        >
```
Don't forget about formatting, check in all places.